src/api/repos/user-repo.ts

```diff
@@ -1,67 +1,71 @@
-import { User } from '../models/User';
-import { httpStatusCode } from '../../types/httpStatusCodes';
+import { User } from '../models';
 import { APIError } from '../../errors/api-error';
 import { BaseError } from '../../errors/base-error';
+import { httpStatusCode } from '../../types/httpStatusCodes';
 import { ValidationErrorItem } from 'sequelize';
 
-// Get all accounts from table account
+/** Get all accounts from table account */
 const getAllUsers = async () => {
     await User.sync();
-    return User.findAll();
+    return User.findAll({
```

Comment:
I think we should add a catch clause here and should probably update the user-repo too.